components/composed/contributor/ContributorCreatePersonForm/ContributorCreatePersonForm.tsx

```diff
@@ -21,8 +22,26 @@ export default function ContributorCreatePersonForm() {
   );
 
   const renderForm = useRenderForm<Fields>(
-    ({ form: { register } }) => (
-      <ContributorPersonForm data={null} register={register} />
+    ({ form: { register, control } }) => (
+      <Forms.Grid>
+        <Forms.Input label="First Name" {...register("givenName")} />
+        <Forms.Input label="Last Name" {...register("familyName")} />
+        <Forms.FileUpload label="Image" name="image" />
+        <Forms.Input label="Title" {...register("title")} />
+        <Forms.Email
+          label="Email"
+          {...register("email")}
+          description="Format: example@email.com"
+        />
+        <Forms.Input label="Affiliation" {...register("affiliation")} />
+        <Forms.Textarea label="Bio" {...register("bio")} />
+        <LinksRepeater<Fields>
```

Comment:
Last bit... now that it's abstracted, this should probably be `Forms.LinksRepeater`. I'm just not sure where that's defined.